manifest.json

```diff
@@ -2,11 +2,11 @@
   "author": "Marco Castelluccio",
   "manifest_version": 2,
   "name": "Gecko Code Coverage",
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
Can you keep the version 0.7.1? We can update it in a separate commit (I'd like to keep the history clean).